.svglintrc.js

```diff
@@ -18,4 +18,13 @@ export default {
       },
     ],
   },
+  ignore: [
+    "build/**",
+    "docs/**",
+    "node_modules/**",
+    "web/locale/**",
+    "*~/**",
+    "test/**",
+    "src/**",
```

Comment:
Can we please include all the same entries, in the same order, as https://github.com/mozilla/pdf.js/blob/master/.eslintignore here?